pyproject.toml

```diff
@@ -14,7 +14,7 @@
 [build-system]
 requires = [
     # We need at least version 36.6.0 that introduced "build_meta"
-    "setuptools>=36.6.0",
+    "setuptools>=36.6.0,<80",
```

Comment:
This is a pretty nasty thing to do to distro packagers. Is this really necessary or can it be worked around?

Reply:
Yeah, I have a fix for this. I'll open a separate pull request.

Reply:
https://github.com/apache/buildstream/pull/2007

Reply:
Yup thanks a lot for that !

Rebased and this commit is gone now.